### DISCUSSION TITLE
Pillowfort.social support

### DISCUSSION THREAD

**Discussion by drobb0690** [TumblThreeApp/TumblThree#328] [Ideas]
I realize this is probably a big stretch and not even on the radar but is there by any chance this could be added?  Or  a separate app made?  Since this is kinda where some of the tumblr folks have migrated.  I would be willing to provide an invitation if needed to the site to help as its currently still in a kinda "closed beta" state.  Thank you and if its not that ok as well.  Thank you for the hard work.

**Comment by thomas694** [comment]
Well, let's see. They are there since five years. They still (or again) have an invite-only access. They say there are more than 100K (registered) users. But what's the active user base? Does anyone have more details about it?

I can see that they have a demo account, but I'm not sure if you really see all. Maybe you can send a link to my email, then I can have another look the next days.
It seems that we could be able to integrate the site. But one problem is, as long as their site isn't public, we would also need to implement some kind of login/authentication for their site.

@ all
Are you also active on their site? Are you interested in an integration?

**Comment by drobb0690** [comment]
I sent an email to the one you have on your profile.  Regardless if anything comes of this thanks for even taking the time.

**Comment by thomas694** [comment]
I got it. Thanks. I'll check it soon.

**Comment by thomas694** [comment]
I've looked at it. It would be possible to do it. But in general I didn't see too much activity there, except a few power users.

The integration couldn't be done by me in the next few weeks. But if someone else likes to give it a try, a PR would be welcome. And I could support by answering one or the other question related to the code.

**Comment by drobb0690** [comment]
Hey thanks a lot for even taking a look.  And I agree it doesn't have a ton of use except by a few users but still think it may be worth it in the long run but that's my opinion as someone who just wants to be able to scrape them, lol. And it if it takes as long as it takes that's ok.   There is no rush
